flink-runtime/src/main/java/org/apache/flink/runtime/util/BashJavaUtils.java

```diff
@@ -33,6 +33,8 @@
  */
 public class BashJavaUtils {
 
+	private static final String EXECUTION_PREFIX = "EXEC_PREFIX:";
```

Comment:
I would use something like "BASH_JAVA_UTILS_EXEC_RESULT: ".
After all, the content following it is a result rather than a prefix.